source/MRVoxels/MRMarchingCubes.cpp

```diff
@@ -946,6 +1057,24 @@ Expected<Mesh> marchingCubes( const FunctionVolume& volume, const MarchingCubesP
     } );
 }
 
+Expected<TriMesh> marchingCubesAsTriMesh( const SimpleBinaryVolume& volume, const MarchingCubesParams& params /*= {} */ )
+{
+    if ( params.iso <= 0 || params.iso >= 1 )
+        return TriMesh{};
```

Comment:
should we have non strict conditions?

Reply:
It is like existing functions in the same file